docs/features.rst

```diff
@@ -61,7 +60,7 @@ considered to be part of the pre-data section of a Postgres dump.
 
 This means that pgcopydb relies on ``pg_dump`` to import the large object
 metadata from the source to the target Postgres server, but then implements
-its own code to migrate the large objects contents, using several worker
+its own logic to migrate the large objects contents, using several worker
```

Comment:
We "implement logic", we "develop code". Or another way to see it is that pgcopydb does not implement anything, we asl developers do "implement" :)

Reply:
In my local dictionary I see the following definition, which makes me think that pgcopydb indeed implements its code into effect, if you consider pgcopydb process at run-time, rather than the git repository. I still think I implement code as soon as the resulting binary is ran, which is the only way my activity makes sense ;-)

> verb | ˈɪmplɪmɛnt | [with object]: put (a decision, plan, agreement, etc.) into effect.